config/sidekiq.yml

```diff
@@ -16,10 +16,10 @@
     every: '5s'
     class: ImmediateEmailGenerationWorker
   daily_digest_initiator:
-    cron: '30 8 * * * Europe/London' # every day at 8:30am
+    cron: '0 9 * * * Europe/London' # every day at 9:00am
     class: DailyDigestInitiatorWorker
   weekly_digest_initiator:
-    cron: '30 8 * * 6 Europe/London' # every Saturday at 8:30am
+    cron: '0 9 * * 6 Europe/London' # every Saturday at 8:30am
```

Comment:
do we want to bump the weekly digest as well?

Reply:
The card only mentioned the daily digest and the weekly one runs on Saturday so I don't think we need to change it. I'll double check!

Reply:
I didn't mean to commit this change btw 🤦‍♂

Reply:
👍

Reply:
Only the daily one needs changing